package.json

```diff
@@ -21,6 +22,9 @@
     }
   },
   "devDependencies": {
+    "@commitlint/cli": "^17.4.3",
```

Comment:
🔶 Up until now, all the dependencies have been pinned to exact versions. I slightly prefer this style of exact versioning, but I'm happy to change my mind if the team prefer otherwise.

Reply:
I prefer ranges because package-lock.json is for locking specific versions. No need to do that in package.json file :)

But yeah, willing to discuss.